GliaWidgets/Glia.swift

```diff
@@ -34,84 +34,6 @@ public protocol SceneProvider: AnyObject {
 }
 
 /// Glia's engagement interface.
-///
```

Comment:
I removed this from here because it was moved to the Integration guide. I think we have agreed on that previously, @kadriiiii  :)